test/pigz/CMakeLists.txt

```diff
@@ -64,11 +64,11 @@ endif()
 FetchContent_Declare(pigz
     GIT_REPOSITORY https://github.com/madler/pigz.git
     GIT_TAG ${PIGZ_TAG})
-FetchContent_MakeAvailable(pigz)
-FetchContent_GetProperties(pigz)
 
+FetchContent_GetProperties(pigz)
 if(NOT pigz_POPULATED)
     FetchContent_Populate(pigz)
+    add_subdirectory(${pigz_SOURCE_DIR} ${pigz_BINARY_DIR})
```

Comment:
This line seems to fail.

Reply:
You are right, it is unneeded because we are using `FetchContent` only to retrieve the source code then we build manually because pigz has no CMakeLists.txt.